tasks/mount.yml

```diff
@@ -52,6 +52,18 @@
   loop: "{{ os_manila_mount_shares | zip(os_manila_mount_share_info) }}"
   loop_control:
     label: "{{ item[0].share_name }}"
+  when: not os_manila_mount_fuse | bool
+
+- name: FUSE-mount the Ceph share
+  ansible.builtin.command:
+    cmd: "ceph-fuse --id {{ item[1].share_user }} \
+      -k {{ os_manila_mount_ceph_conf_path }}/ceph.client.{{ item[1].share_user }}.keyring \
+      -r {{ item[1].export }} \
+      {{ item[0].mount_path }}"
+  loop: "{{ os_manila_mount_shares | zip(os_manila_mount_share_info) }}"
+  loop_control:
+    label: "{{ item[0].share_name }}"
+  when: os_manila_mount_fuse | bool
```

Comment:
In theory, we should be able to use `ansible.posix.mount` with `fstype: fuse.ceph`. I've not yet been able to get that working, would see the following error.

```
sudo mount -t fuse.ceph -o ceph.id=cluster-prod,ceph.conf=/etc/ceph/ceph.conf,ceph.client-mountpoint=/volumes/_nogroup/<id-omitted>/<different-id-omitted> /scratch -v
2025-07-02T10:52:57.657+0000 7f712b113480 -1 init, newargv = 0x5645987c1070 newargc=15
2025-07-02T10:52:57.657+0000 7f712b113480 -1 init, args.argv = 0x564598905510 args.argc=4
ceph-fuse[103540]: starting ceph client
fuse: unknown option(s): `-o name=cluster-prod'
ceph-fuse[103540]: fuse failed to start
2025-07-02T10:53:40.883+0000 7f712b113480 -1 fuse_session_new failed
Mount failed with status code: 33
```

Reply:
Got this working, I was hitting the above error due to stale data in `/etc/fstab`